tests/conftest.py

```diff
@@ -18,3 +21,32 @@ def enable_db_access_for_all_tests(
     A pytest fixture that enables database access for all tests.
     """
     pass
+
+
+@pytest.fixture(scope="session", autouse=True)
+def faker_session_locale():
+    """
+    A pytest fixture that returns the locale for Faker.
+    """
+    return ["ru_RU"]
+
+
+@pytest.fixture
+def fake_personal_info_data(faker):
+    return {
+        "communication_language_mode": faker.random_element(
+            PersonalInfo.CommunicationLanguageMode.values
+        ),
+        "first_name": faker.first_name(),
+        "last_name": faker.last_name(),
+        "tg_username": faker.user_name(),
+        "email": faker.email(),
+        "phone": faker.numerify("+791########"),
+        "utc_timedelta": "03:00:00",
+        "information_source": faker.pystr(),
```

Comment:
maybe use a [Lorem](https://faker.readthedocs.io/en/master/providers/faker.providers.lorem.html) for this?

Reply:
Seems good. Fixed to .text()